frontend/src/components/organisms/sidebar/Sidebar.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import styled from 'styled-components';
+import { NavLink } from 'react-router-dom';
+import LogoImage from '../../../assets/logo.png';
+import IconButton from '../../atoms/IconButton/IconButton';
+import { ReactComponent as IconAlarm } from '../../../assets/svgs/alarm-light.svg';
+import theme from '../../../theme/MainTheme';
+
+const MenuWrapper = styled.div`
+  display: grid;
+  row-gap: 10px;
+  justify-items: start;
+`;
+
+const LogoWrapper = styled.div`
+  display: inline-grid;
+  grid-template-columns: 250px auto;
+  justify-items: center;
+  align-items: center;
+  width: 100%;
+  height: 100px;
+  margin-bottom: 15px;
+`;
+const SpaceLine = styled.hr`
+  height: 1px;
+  width: 100%;
+  background-color: ${theme.grey400};
+  border: none;
+  margin: 40px 0;
+`;
+
+const Wrapper = styled.nav`
```

Comment:
Whole sidebar should have color: #FBFBFB

Reply:
ok